app/src/main/java/com/perflyst/twire/fragments/StreamFragment.java

```diff
@@ -1280,6 +1280,7 @@ private void pauseStream() {
 
         if (isAudioOnlyModeEnabled()) {
             Log.d(LOG_TAG, "Pausing audio");
+            player.pause();
         } else if (player != null) {
             player.pause();
         }
```

Comment:
I think this can be simplified to:
```suggestion
        if (player != null) {
            player.pause();
        }
```

Reply:
Changed in https://github.com/twireapp/Twire/pull/229/commits/f1eeaa347d78c10662b394e8ac659fac59761445